deployment/docker/docker-compose.yml

```diff
@@ -287,7 +287,9 @@ services:
       GUNICORN_WORKERS: ${GUNICORN_WORKERS:-5}
       GUNICORN_THREADS: ${GUNICORN_THREADS:-10}
       GUNICORN_TIMEOUT: ${GUNICORN_TIMEOUT:-120}
-      
+      MULTITENANT_BPM_CLIENT_ID: ${MULTITENANT_BPM_CLIENT_ID}
+      MULTITENANT_BPM_CLIENT_SECRET: ${MULTITENANT_BPM_CLIENT_SECRET}
```

Comment:
If MULTI_TENANCY_ENABLED is true, isn't providing the multitenant client secret in KEYCLOAK_BPM_CLIENT_SECRET enough?

Reply:
@auslin-aot, this is not the one used in KEYCLOAK_BPM_CLIENT_SECRET. its used to deploy the tenant specific Camunda workflow, which is triggered after the tenant registration. I have added commented in sample.env. let me know, if you prefer any other meaningful name for the variables